### DISCUSSION TITLE
Please add "visual settings" while paused so that players can easy adjust.

### DISCUSSION THREAD

**Discussion by NamOpt** [ppy/osu#26278] [Ideas] [answered]
The comparison between lazer and stable:
lazer paused menu:
https://github.com/ppy/osu/assets/120299835/67dd5a97-5c9b-4d7c-91bf-18bdb24defc9

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/3853.
